### PR TITLE
Add Makefile targets for kafka, namespace and teardown

### PR DESCRIPTION
**Pull request by jchesterpivotal**
Intended to make the development experience more consistent.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -50,3 +66,4 @@ clean:
 	$(MAKE) -C topic-controller		clean
 	$(MAKE) -C riff-cli				clean
 
+dev-setup: namespace development-kafka clean build dockerize kubectl-apply
```
Comment (ericbottard) [inline Makefile]: I'm not too keen on so many fine grained tasks that are only aliases for one kubectl command. What about moving the namespace and kafka parts inside kubectl-apply?

Comment (jchesterpivotal) [reply Makefile]: I don't mind at all. I figured I'd start with a straight translation.

Would it be worth renaming `kubectl-apply` in that case? (To what I'm not sure, as I haven't had my 2nd coffee yet)

Comment (ericbottard) [reply Makefile]: `dev-setup`? then it neatly mirrors `teardown`

Comment (jchesterpivotal) [reply Makefile]: Works for me. I'll push it up in a few.

File: Makefile [hunk 2]
```diff
@@ -30,13 +30,25 @@ debug-dockerize:
 	$(MAKE) -C http-gateway			debug-dockerize
 	$(MAKE) -C topic-controller		debug-dockerize
 
-kubectl-apply:
+dev-setup: clean dockerize
```
Comment (ericbottard) [inline Makefile]: I wouldn't put clean dockerize as dependencies.

dev-setup and teardown need to be declared as PHONY

**Comment by ericbottard** [comment]
Merged as 75bd4bb3

**Pull request closed without merging**